sdk/communication/communication-common/src/identifierModels.ts

```diff
@@ -206,6 +259,30 @@ export const getIdentifierRawId = (identifier: CommunicationIdentifier): string
       }
       return `8:orgid:${microsoftTeamsUserId}`;
     }
+    case "microsoftTeamsBot": {
+      const { microsoftTeamsBotId, rawId, cloud, isGlobal } = identifierKind;
+      if (rawId) return rawId;
+      if (isGlobal) {
+        switch (cloud) {
+          case "dod":
+          return `28:dod-global:${microsoftTeamsBotId}`;
+        case "gcch":
+          return `28:gcch-global:${microsoftTeamsBotId}`;
+        case "public":
+          return `28:${microsoftTeamsBotId}`;
+        }
+        return `28:${microsoftTeamsBotId}`;
+      }
+      switch (cloud) {
+        case "dod":
+          return `28:dod:${microsoftTeamsBotId}`;
+        case "gcch":
+          return `28:gcch:${microsoftTeamsBotId}`;
+        case "public":
+          return `28:orgid:${microsoftTeamsBotId}`;
+      }
+      return `28:orgid:${microsoftTeamsBotId}`;
+    }
```

Comment:
Similar to above comment.
Instead of duplicating code on line 282 and 284, we can have a default case
Line 281 : 
```
default :
   return `28:orgid:${microsoftTeamsBotId}`;
```

Reply:
Thanks, updated!